ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -154,6 +154,12 @@
         output_dir: "{{ cifmw_edpm_prepare_basedir }}/artifacts"
         script: "oc apply -f {{ cifmw_edpm_prepare_openstack_cr_manifest_paths.files[0].path }}"
 
+    - name: Override openstack service images
+      when:
+      - cifmw_openstack_svc_image is defined
```

Comment:
Where does this variable come from? It's not part of a role and it's not documented as a global one.

Reply:
When the images are collected from an existing cluster, it is stored in a yaml file under this key.